dev/ctests/cases/C48_S2SW_gfs_fcst_seg0.yaml

```diff
@@ -7,44 +7,541 @@
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f108.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f108.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f109.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f109.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f110.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f110.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f111.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f111.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f1{{ cyc }}.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f1{{ cyc }}.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f113.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f113.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f114.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f114.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f115.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f115.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f116.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f116.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f117.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f117.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f118.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f118.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f119.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f119.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f{{ cyc }}0.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f{{ cyc }}0.bin]
+        - [{{ SRC_DIR }}/logs/{{ PDY }}{{ cyc }}/gfs_fcst_seg0.log, {{ DST_DIR }}/logs/{{ PDY }}{{ cyc }}/gfs_fcst_seg0.log]
```

Comment:
log file won't match.

Reply:
We know of course.  They are here for completeness.  The ***_cmp** files where for comparisons validation which we will address when we include subsections per type.